docs/domains/Domain.json

```diff
@@ -499,6 +499,10 @@
             "$ref": "https://github.com/garethr/kubernetes-json-schema/blob/master/v1.13.5/_definitions.json#/definitions/io.k8s.api.core.v1.PodReadinessGate"
           }
         },
+        "serviceAccountName": {
+          "description": "ServiceAccountName is the name of the ServiceAccount to use to run this pod. If it is not set, default ServiceAccount will be used. The ServiceAccount has to exist at the time the pod is created.",
```

Comment:
Since this is already the description, it doesn't need to be self-referential.  "ServiceAccountName is the name of the ServiceAccount..." -> "Name of the ServiceAccount..."

Reply:
fixed